server/backend/testspg/router/test_router_application.py

```diff
@@ -244,4 +257,243 @@ def test_fam_application_endpoints_invlid_path_application_id_type(
+    # and FORM_SUBMITTER_00001018, which is granted in the local sql. It also
+    # is the same organization as "LOAD-3-TEST" user but not the same
+    # organization as "LOAD-4-TEST".
+    token = jwt_utils.create_jwt_token(
+        test_rsa_key, roles=[], username=jwt_utils.COGNITO_USERNAME_BCEID
+    )
+    fom_dev_bceid_dgadmin_user_access_grants_response = test_client_fixture.get(
+        get_role_assignment_end_point, headers=jwt_utils.headers(token)
+    )
+    fom_dev_bceid_dgadmin_rs_data = fom_dev_bceid_dgadmin_user_access_grants_response.json()
+    # this BCEID delegated admin should only see user access granted for
+    # FOM_DEV FOM_REVIEWER and FORM_SUBMITTER_00001018 role for BCEID users,
+    # within its organization, not FORM_SUBMITTER_00000001 or
+    # FORM_SUBMITTER_00001011.
+    fom_dev_bceid_access_grants_for_reviewer_l3t = [
+        grant for grant in fom_dev_bceid_reviewer_access_grants
+        if grant["role_id"] == FOM_DEV_REVIEWER_ROLE_ID and
+        grant["user_name"] == USER_NAME_BCEID_LOAD_3_TEST
+    ]
+    fom_dev_bceid_submitters_00001018_access_grants_l3t = [
+        grant for grant in fom_dev_bceid_submitters_00001018_access_grants
+        if grant["role_id"] == FOM_DEV_SUBMITTER_ROLE_ID and
+        grant["user_name"] == USER_NAME_BCEID_LOAD_3_TEST and
+        grant["forest_client_number"] == FC_NUMBER_EXISTS_ACTIVE_00001018
+    ]
+    assert len(fom_dev_bceid_dgadmin_rs_data) == (
+        len(fom_dev_bceid_access_grants_for_reviewer_l3t) +
+        len(fom_dev_bceid_submitters_00001018_access_grants_l3t)
+    )
+    assert all(
+        granted["role"]["application_id"] == FOM_DEV_APPLICATION_ID and
+        (granted["role"]["role_name"] == ROLE_NAME_FOM_REVIEWER or
+         granted["role"]["role_name"] == ROLE_NAME_FOM_SUBMITTER_00001018) and
+        granted["user"]["user_type_code"] == UserType.BCEID and
+        granted["user"]["user_name"] == USER_NAME_BCEID_LOAD_3_TEST
+        for granted in fom_dev_bceid_dgadmin_rs_data
+    )
+    assert len([granted for granted in fom_dev_bceid_dgadmin_rs_data if (
+        granted["role"]["application_id"] == FOM_DEV_APPLICATION_ID and
+        granted["user"]["user_type_code"] == UserType.BCEID and
+        granted["role"]["role_name"] == ROLE_NAME_FOM_REVIEWER and
+        granted["user"]["user_name"] == USER_NAME_BCEID_LOAD_3_TEST
+    )]) == len(fom_dev_bceid_access_grants_for_reviewer_l3t)
+    assert len([granted for granted in fom_dev_bceid_dgadmin_rs_data if (
+        granted["role"]["application_id"] == FOM_DEV_APPLICATION_ID and
+        granted["user"]["user_type_code"] == UserType.BCEID and
+        granted["role"]["role_name"] == ROLE_NAME_FOM_SUBMITTER_00001018 and
+        granted["user"]["user_name"] == USER_NAME_BCEID_LOAD_3_TEST
+    )]) == len(fom_dev_bceid_submitters_00001018_access_grants_l3t)
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
maybe we don't need these many new lines?

Reply:
Hm.... what's going on.... I didn't remember I purposely hit enter so many times.... but maybe I was almost falling asleep when writing a test.... sorry, will remove that. Thanks for noticing it!!!